docs/source/components/nodes/video_encoder.rst

```diff
@@ -77,6 +77,7 @@ Usage
 Limitations
 ###########
 
+- HW limit for the encoder is: 3840x2160 pixels at 30FPS or :code:`248 million pixels/second`. The resolution and frame rate can be divided up to 3 streams - but the sum of all the pixels/second needs to be below 248 million.
```

Comment:
It's actually possible to have more than 3 streams, as long as we don't exceed the 4K 30fps capability.
Here is H.264 encoding 4x cameras on OAK-FFC-4P, each at 1080p 30fps: https://github.com/alex-luxonis/depthai-python/commit/e2342c8